examples/diabetes_example/run_diabetes.py

```diff
@@ -0,0 +1,500 @@
+    assets_directory / "functions" / "aggregate_function_analytics.zip"
+)
+with zipfile.ZipFile(aggregate_archive_path, "w") as z:
+    for filepath in aggregate_function_docker_files:
+        z.write(filepath, arcname=os.path.basename(filepath))
+
+aggregate_function_inputs = [
+    FunctionInputSpec(
+        identifier="local_analytics_list",
+        kind=AssetKind.model,
+        optional=False,
+        multiple=True,
+    ),
+]
+
+aggregate_function_outputs = [
+    FunctionOutputSpec(identifier="shared_states", kind=AssetKind.model, multiple=False)
+]
+
+aggregate_function = FunctionSpec(
+    name="Aggregate Federated Analytics",
+    inputs=aggregate_function_inputs,
+    outputs=aggregate_function_outputs,
+    description=assets_directory / "functions" / "description.md",
+    file=aggregate_archive_path,
+    permissions=permissions,
+)
+
+
+aggregate_function_key = clients[ALGO_ORG_ID].add_function(aggregate_function)
+
+print(f"Aggregation function key {aggregate_function_key}")
+
+# %%
+# Second local step
+# -----------------
+# Idem for the second round of computations happening locally on the data nodes.
+#
+# Both aggregation steps will use the same function, so we don't need to register it again.
+#
+# %%
+
+# %load -s local_step2 assets/functions/federated_analytics_functions.py
+
+# %%
+
+local_function_2_docker_files = [
+    assets_directory / "functions" / "federated_analytics_functions.py",
+    assets_directory / "functions" / "local_compute_2" / "Dockerfile",
+]
+
+local_archive_2_path = assets_directory / "functions" / "local_function_analytics.zip"
+with zipfile.ZipFile(local_archive_2_path, "w") as z:
+    for filepath in local_function_2_docker_files:
+        z.write(filepath, arcname=os.path.basename(filepath))
+
+local_function_2_inputs = [
+    FunctionInputSpec(
+        identifier="datasamples",
+        kind=AssetKind.data_sample,
+        optional=False,
+        multiple=True,
+    ),
+    FunctionInputSpec(
+        identifier="opener", kind=AssetKind.data_manager, optional=False, multiple=False
+    ),
+    FunctionInputSpec(
+        identifier="shared_states", kind=AssetKind.model, optional=False, multiple=False
+    ),
+]
+
+local_function_2_outputs = [
+    FunctionOutputSpec(
+        identifier="local_analytics_second_moments",
+        kind=AssetKind.model,
+        multiple=False,
+    )
+]
+
+local_function_2 = FunctionSpec(
+    name="Local Federated Analytics - step 1",
+    inputs=local_function_2_inputs,
+    outputs=local_function_2_outputs,
+    description=assets_directory / "functions" / "description.md",
+    file=local_archive_2_path,
+    permissions=permissions,
+)
+
+
+local_function_2_keys = {
+    client_id: clients[client_id].add_function(local_function_2)
+    for client_id in DATA_PROVIDER_ORGS_ID
+}
+
+print(f"Local function key for step 2 {local_function_2_keys}")
+
+# %%
+# The data and the functions are now registered.
+#
+
+# %%
+# Registering tasks
```

Comment:
Let's put it on the same heading level as Adding Function? And same wording either Add or Register for both.